README project description

Alvarado08 opened the pull request:
Hello, team! Could you kindly check the current project content and structure of our README? Just to make sure we're on the same page. Feel free to leave any comments or suggestions.

Alvarado08 committed:
README project description

KevinLlano reviewed (changes requested):
Looks good, but it’s showing a merge conflict, so you should resolve that before I approve.

Tayrika commented on README.md:
```diff
@@ -1,40 +1,109 @@
+
+## Contributing
+
+We welcome contributions to Animal Shelter! Please follow these steps:
```
Animal Shelter -> Chingu Demographics

Tayrika commented on README.md:
```diff
@@ -1,40 +1,109 @@
+
+## License
+
+This project is licensed under the [MIT License](https://github.com/chingu-voyages/V54-tier3-team-36/blob/development/LICENSE).
```
This can be changed to the link of V58-tier3-team-39 :)

Tayrika commented on README.md:
```diff
@@ -1,40 +1,109 @@
+
+## Team Members
+
+This project was developed by an amazing team as part of Chingu Voyage 54:
```
Voyage 54 -> Voyage 58

Alvarado08 committed:
README fixes

Removed License section and AI Chatbot feature. Added a Tools section under Technologies Used. Updated team info.

Valeriusdev reviewed (changes requested):
please resolve a merge conflict, when you have a chance

Alvarado08 committed:
README changes

Alvarado08 committed:
Merge branch 'dev' into feature/update-readme
